libqtile/backend/wayland/cffi/build.py

```diff
@@ -1,3 +1,9 @@
+import sys
+from pathlib import Path
+
+path = Path(__file__).parent / ".." / ".." / ".." / ".."
```

Comment:
wut?

Reply:
Ha!

So, as far as I can tell, there was an issue when trying to build this as I got a modulenotfound error for qtile.

This line is just working out the path for the root folder (i.e. the folder containing libqtile). `Path(__file__)` gives the path to `build.py`, `.parent` is used to get the folder and then you can use the `/ ".."` syntax as if it were a file path to move up levels.